### PR TITLE
docs(fix): code blocks on light theme

### PR DESCRIPTION
**Pull request by userquin**
I'm talking with brc-dd to try to fix this on VP... we can remove/update styles added in this PR once we've a way to configure light code theme on VP

### CODE REVIEW

File: docs/.vitepress/theme/style.css [hunk 1]
```diff
@@ -115,3 +120,41 @@
   transition: color 0.25s;
 }
 
+.vp-doc th,
```
Comment (antfu) [inline docs/.vitepress/theme/style.css:123]: Thanks! But this feels a bit too much. I rather wait for VP to fix it.

Comment (userquin) [reply docs/.vitepress/theme/style.css:123]: I'll try to simplify it a bit, I just copy styles from pwa docs, we are not using a lot of cases (code blocks inside details)

Comment (antfu) [reply docs/.vitepress/theme/style.css:123]: Can we keep the complexity in VP? I think we should try to avoid duplications across repos

Comment (userquin) [reply docs/.vitepress/theme/style.css:123]: Sure

Comment (userquin) [reply docs/.vitepress/theme/style.css]: Anthony, simplified to minimal changes

**Pull request merged**